src/js/themes/base.js

```diff
@@ -886,6 +886,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       },
       gap: 'small',
       size: `${baseSpacing}px`,
+      // fontWeight: undefined,
```

Comment:
Great, thanks for adding this line.
Could you also add the prop to[ base.d.ts](https://github.com/grommet/grommet/blob/7ed818375486b580b9585a2bd9d5ad90037a1a52/src/js/themes/base.d.ts#L836), similarly to how other font weights are defined?

Reply:
Will do!